src/view/documentfragment.js

```diff
@@ -20,6 +20,9 @@ export default class DocumentFragment {
 	/**
 	 * Creates new DocumentFragment instance.
 	 *
+	 * **Note:** Constructor of this class shouldn't be used directly in the code. Use the
+	 * {@link module:engine/view/upcastwriter~UpcastWriter#createDocumentFragment} method instead.
```

Comment:
```suggestion
	 * {@link module:engine/view/upcastwriter~UpcastWriter#createDocumentFragment `UpcastWriter#createDocumentFragment()`}
	 * method instead.
```

Reply:
Please repeat that change in other places as well. Without the additional link text, such a link generates as a useless `createDocumentFragment`. Without clicking it you don't know where to find it. Plus, it does not render with `()` so without reading the text it does not look like a method.